ethereum/contracts/bridge/Bridge.sol

```diff
@@ -20,6 +20,18 @@ import "./token/TokenImplementation.sol";
 contract Bridge is BridgeGovernance, ReentrancyGuard {
     using BytesLib for bytes;
 
+    /**
+     * @notice Emitted when a transfer is completed by the token bridge.
+     * @param emitterChainId Wormhole chain ID of emitter on the source chain.
+     * @param emitterAddress Address (bytes32 zero-left-padded) of emitter on the source chain.
+     * @param sequence Sequence of the Wormhole message.
+     */
+    event TransferRedeemed(
```

Comment:
naming nit: wouldn't `TransferCompleted` be a more appropriate name, since it's emitted by the `completeTransfer*` family of functions?

Reply:
The Sui contract emits an event named `TransferCompleted` already, I figured we should be consistent. Although, I agree that's a better name.